common/utils/query_utils.ts

```diff
@@ -9,12 +9,45 @@
  * GitHub history for details.
  */
 
-import { PPL_INDEX_REGEX } from "../constants/shared";
+import { isEmpty } from 'lodash';
+import datemath from '@elastic/datemath';
+import { DATE_PICKER_FORMAT } from '../../common/constants/explorer';
+import {
+  PPL_INDEX_REGEX,
+  PPL_INDEX_INSERT_POINT_REGEX
+} from '../../common/constants/shared';
 
 export const getIndexPatternFromRawQuery = (query: string) : string => {
   const matches = query.match(PPL_INDEX_REGEX);
   if (matches) {
     return matches[2];
   }
   return '';
+};
+
+export const insertDateRangeToQuery = ({
+  rawQuery,
+  startTime,
+  endTime,
+  timeField = 'utc_time',
+}: {
+  rawQuery: string;
+  startTime: string;
+  endTime: string;
+  timeField?: string;
+}) => {
+
+  let finalQuery = '';
+
+  if (isEmpty(rawQuery)) return finalQuery;
+    
+  // convert to moment
+  const start = datemath.parse(startTime)?.format(DATE_PICKER_FORMAT);
+  const end = datemath.parse(endTime)?.format(DATE_PICKER_FORMAT);
```

Comment:
if you select something like `Today` or `This week` from date picker's "commonly used", i remember `startTime` and `endTime` will be the same string. could you check if you need to do some handling for that

Reply:
Sure, this makes sense, I'll change it.